lib/registration.pm

```diff
@@ -211,9 +211,12 @@ sub register_addons_cmd {
         elsif ($name =~ /LTSS/) {
             add_suseconnect_product($name, undef, undef, "-r " . get_var('SCC_REGCODE_LTSS'));
         }
-        else {
+        elsif ($name =~ /sdk/) {
             add_suseconnect_product($name);
         }
+        else {
```

Comment:
This branch could be ignore ?

Reply:
Yes, this is only for those particular modules. Some other module like HA, don't need re-register.